### PR TITLE
added sample python notebook for composite data readers

### PR DESCRIPTION
**Pull request by drcrook1**

### CODE REVIEW

**Comment by drcrook1** [comment]
Not sure I created pull request properly; feel free to let me know appropriate way if I did it wrong.

**Comment by sayanpa** [comment]
please remove the checkpoint files.
Create a directory under examples called Multimodal. Inside it create a dir describing the data_domain and data_type in the name.
Also we will need to add a test under \Tests\EndToEndTests\CNTKv2Python\Tutorials

**Comment by drcrook1** [comment]
Righto.  Can you specify exactly what you want for the directory it should go into?  I'm not 100% following what you want.  I'll take a look at how to do a test as well.  Should have something tomorrow.

**Comment by drcrook1** [comment]
Alright, I left 1 .pynb checkpoint; because it shows things working and rendered out.  If you want that one removed; let me know, I'll need to add some additional code blocks to showcase what I want.

I also gave a shot at naming the folder something that made sense.  Let me know what you think.

**Comment by drcrook1** [comment]
Anything else I need to do before getting this merged in?

File: Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+    "    return loss, errs # (model, labels) -> (loss, error metric)        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "## Training Progress Printer\n",
```
Comment (sayanpa) [inline Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb]: Would prefer to use ProgressPrinter  framework. Refer to https://github.com/Microsoft/CNTK/blob/master/Manual/Manual_How_to_feed_data.ipynb for example

File: Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+    "    } \n",
+    "    \n",
+    "    training_progress_output_freq = 500\n",
+    "     \n",
+    "    # Start a timer\n",
+    "    start = time.time()\n",
+    "\n",
+    "    for i in range(0, int(num_minibatches_to_train)):\n",
+    "        # Read a mini batch from the training data file\n",
+    "        data=train_reader.next_minibatch(minibatch_size, input_map=input_map)         \n",
```
Comment (sayanpa) [inline Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb]: can we please use training_session: refer to https://github.com/Microsoft/CNTK/blob/master/Manual/Manual_How_to_feed_data.ipynb

File: Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Understanding The Output\n",
+    "\n",
+    "**Errors**\n",
+    "The average error is important to understand as it helps understand across all possible outputs what your error is.  I also like to be able to break the average error back into the same numerical space as the question.  Here we basically just undo the MSE and assume same error across all nodes and calculate RMSE as it would exist per output node.  Basically, you can think of this as on average each node is going to be + or - 0.52.  So if the actual value is 2, you are likely to predict a value roughly around 1.48 or 2.52 on average.\n",
+    "\n",
+    "**Trends in Errors**\n",
```
Comment (sayanpa) [inline Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb]: This section is very confusing. Would appreciate re-wording this section.

File: Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb [hunk 4]
```diff
@@ -0,0 +1,429 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Composite Data Readers Tutorial**\n",
```
Comment (sayanpa) [inline Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb]: Remove the checkpoint file please. Also, I left a lot of comments with the checkpoint file that needs to be incorporated into the actual ipynb file.

File: Examples/MultiModal/Mixed_Image__And_Tabular_Reading/CompositeReader.ipynb [hunk 5]
```diff
@@ -0,0 +1,429 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Composite Data Readers Tutorial**\n",
```
Comment (sayanpa) [inline Examples/MultiModal/Mixed_Image__And_Tabular_Reading/CompositeReader.ipynb]: This notebook is looking pretty good. We would like to merge this with the https://github.com/Microsoft/CNTK/blob/master/Manual/Manual_How_to_feed_data.ipynb

Could you please make the changes?

Comment (drcrook1) [inline Examples/MultiModal/Mixed_Image__And_Tabular_Reading/CompositeReader.ipynb]: I'll have to do this after September as I am bogged down until then.

**Comment by sayanpa** [comment]
Sounds good David. Will close this one for now, please reopen when you are ready.

**Pull request closed without merging**